scene/gui/graph_frame.h

```diff
@@ -0,0 +1,108 @@
+/**************************************************************************/
+/*  graph_frame.h                                                         */
+/**************************************************************************/
+/*                         This file is part of:                          */
+/*                             GODOT ENGINE                               */
+/*                        https://godotengine.org                         */
+/**************************************************************************/
+/* Copyright (c) 2014-present Godot Engine contributors (see AUTHORS.md). */
+/* Copyright (c) 2007-2014 Juan Linietsky, Ariel Manzur.                  */
+/*                                                                        */
+/* Permission is hereby granted, free of charge, to any person obtaining  */
+/* a copy of this software and associated documentation files (the        */
+/* "Software"), to deal in the Software without restriction, including    */
+/* without limitation the rights to use, copy, modify, merge, publish,    */
+/* distribute, sublicense, and/or sell copies of the Software, and to     */
+/* permit persons to whom the Software is furnished to do so, subject to  */
+/* the following conditions:                                              */
+/*                                                                        */
+/* The above copyright notice and this permission notice shall be         */
+/* included in all copies or substantial portions of the Software.        */
+/*                                                                        */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,        */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF     */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. */
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY   */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,   */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE      */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
+/**************************************************************************/
+
+#ifndef GRAPH_FRAME_H
+#define GRAPH_FRAME_H
+
+#include "scene/gui/graph_element.h"
+
+class HBoxContainer;
+
+class GraphFrame : public GraphElement {
+	GDCLASS(GraphFrame, GraphElement);
+
+	struct _MinSizeCache {
```

Comment:
Why is the name prefixed?

Reply:
Was copied from box_container.cpp, I thought this was a convention for private, single-use structs.

Reply:
BoxContainer code is super old and was probably never updated.
Though I don't recall any other case of such struct.

Reply:
I just did a quick search and it looks like prefixing structs defined in cpp files (or private inner structs) with _ seems to be common practice in our code base, even in newer code.

Reply:
Huh. Then it's used inconsistently 🤔